kuka_iiqka_eac_driver/CMakeLists.txt

```diff
@@ -17,8 +17,7 @@ endif()
 # A few (transitive) dependencies are available only in one of the dependencies
```

Comment:
please also remove this comment, as the line it was related to, was removed

Reply:
done